src/main/java/com/jfeatures/msg/sql/ReadFileFromResources.java

```diff
@@ -101,17 +96,11 @@ public static Properties readPropertiesFromFile(String propertyFileName) throws
         return prop;
     }
 
-    public static String readFileFromResources(String fileName) throws URISyntaxException
-    {
-        URL resource = ReadFileFromResources.class.getResource(fileName);
-        assert resource != null;
-        Path path = Path.of(resource.toURI());
-        String fileContent = "";
-        try {
-            fileContent = new String(Files.readAllBytes(path));
+    public static String readFileFromResources(String fileName) {
+        try (var inputStream = ReadFileFromResources.class.getClassLoader().getResourceAsStream(fileName)) {
+            return new String(inputStream.readAllBytes());
         } catch (IOException e) {
-            log.severe("Error reading file from resources" + e.getMessage());
+            throw new UncheckedIOException(e);
```

Comment:
This is good addition, isn't it good idea to keep a logger statement as well? that gives a nice English language message about error.

Reply:
What do you think of the following?

```java
throw new UncheckedIOException("Error reading file from resources: " + e.getMessage(), e);
```

This way we log only one friendly message that includes the cause.
I pushed that. If you think it's better leaving the severe message, just let me know.

Reply:
What do you think of the following?

```java
throw new UncheckedIOException("Error reading file from resources: " + e.getMessage(), e);
```

This way we log only one friendly message that includes the cause.
I pushed that. If you think it's better leaving the severe message, just let me know.